WIP-CHANGELOG.md

```diff
@@ -20,3 +20,4 @@
 
 - Kibana OIDC logout not redirecting correctly.
 - Getting stuck at selecting tenant when logging in to Kibana.
+- Elasticsearch backups failing due to mismatching config in backup job and elasticsearch
```

Comment:
This should not be necessary to mention since it wasn't broken in the last tagged release.
See [v0.6.0](https://github.com/elastisys/ck8s-apps/blob/v0.6.0/config/config/sc-config.yaml#L172) and [the PR where it was added](https://github.com/elastisys/ck8s-apps/commit/42788ed66871e1ad0786ad14b12b77d38b393e70#diff-32ea2de1ac9797b598bd4286c8dde58bfe56155405e078907360a224a977085aR184)